lib/language_pack/ruby.rb

```diff
@@ -383,7 +361,15 @@ def setup_language_pack_environment(ruby_layer_path = ".", gem_layer_path = ".")
       gem_path = "#{gem_layer_path}/#{slug_vendor_base}"
       ENV["GEM_PATH"] = gem_path
       ENV["GEM_HOME"] = gem_path
-      ENV["PATH"]     = default_path(gem_layer_path)
+
+      paths = []
+      paths << "#{File.expand_path(".")}/bin" unless ruby_version.ruby_192_or_lower? # For Ruby 1.9.2 and lower there is a "build" and non-"build" Ruby
```

Comment:
Do you mind copying over the whole comment here from before?

Reply:
Seems good, done.